addons/api/addon/models/target.js

```diff
@@ -2,6 +2,11 @@ import GeneratedTargetModel from '../generated/models/target';
 import { attr } from '@ember-data/model';
 import { inject as service } from '@ember/service';
 
+export const types = {
```

Comment:
It wasn't obvious to me that the array was names for the icons at first, could we maybe rename the variable to something more descriptive to indicate that?

Another question: is there a reason it's an array?

Reply:
It's not really just for icons.  It could be for other stuff and we use a similar concept elsewhere.  I think we need some standardisation around the naming and structure of type hashes.

Reply:
Right, I think it's a little misleading to just call it `types` when usually our other `types` are just arrays for this. 

I think we can keep the types variable just for the array and have a new one for icons that builds off that variable.

Reply:
I agree. I'll look into standardizing the name and the structure